app/javascript/validate-paired-tubes/components/ValidatePairedTubes.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <lb-page>
+    <lb-loading-modal
+      v-if="loading"
+      :message="progressMessage"
+    />
+    <lb-main-content>
+      <lb-transfer-volumes
+        :purpose-configs="purposeConfigs"
+        :tube="sourceTube.labware"
+        :confirmed-pair="allValid"
+      />
+    </lb-main-content>
+    <lb-sidebar>
+      <b-card
+        header="Validate Tubes"
+        header-tag="h3"
+      >
+        <p class="tv-instructions">
+          Scan the LTHR-384 Pool XP tube and the corresponding LB Lib Pool Norm tube. If they match, the transfer volumes will be shown to the left.
```

Comment:
You've hardcoded the tube names here, but the button will appear whenever tubes have the volume parameters configured. Given how the page is currently accessed, I don't think its possible to make the message dynamic (without passing in additional parameters to the initial request) so maybe just make the message a little less specific.